### DISCUSSION TITLE
let replace-query action takes n-th argument

### DISCUSSION THREAD

**Discussion by Kyle0923** [junegunn/fzf#3716] [Q&A]
the `replace-query` action takes whole line of the current selection to replace the query string with it.
I am using it with the rg-fzf integration and the selection line is of the form `FILE:line_num:content` and I only want to use the content part of the line as my query string.
Is there a way to ask `replace-query` to take the n-th argument?
I have tried
```
replace-query({3..})
replace-query(3..)
```

Also tried `change-query({3..})`. But `{3..}` doesn't get expanded in `change-query`.
So I the work around I had is `transform(echo \"change-query(\$(echo {2..}))\")` which is a bit long

**Comment by LangLangBart** [comment]
> Is there a way to ask `replace-query` to take the n-th argument?

`transform-query`[^1]

```bash
echo {13..10} | fzf --bind 'load:transform-query:echo {-1}'
```

[^1]: [fzf/CHANGELOG.md 0.36.0](https://github.com/junegunn/fzf/blob/master/CHANGELOG.md#0360)
